deployment/sriov-network-operator/Chart.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: v2
 name: sriov-network-operator
 version: 0.1.0
-kubeVersion: '>= 1.16.0'
+kubeVersion: '>= 1.16.0-0'
```

Comment:
mind updating kube version to follow k8s release support https://kubernetes.io/releases/

that is 1.26 and newer ?